geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/serial/SerialWANPropagation_PartitionedRegionDUnitTest.java

```diff
@@ -78,6 +78,41 @@ public void testPartitionedSerialPropagation() {
     vm2.invoke(() -> WANTestBase.validateRegionSize(getTestMethodName() + "_PR", 1000));
   }
 
+  @Test
+  public void testPartitionedSerialPropagationWithTXWhenSendersNotConfiguredOnAllServers() {
+
+    Integer lnPort = vm0.invoke(() -> WANTestBase.createFirstLocatorWithDSId(1));
+    Integer nyPort = vm1.invoke(() -> WANTestBase.createFirstRemoteLocator(2, lnPort));
```

Comment:
You can just declare things like `lnPort` as `int` primitive instead of `Integer`.